src/Features/CSharp/Portable/Completion/KeywordRecommenders/DataKeywordRecommender.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp.Extensions.ContextQuery;
+using Microsoft.CodeAnalysis.CSharp.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Completion.KeywordRecommenders
+{
+    internal class DataKeywordRecommender : AbstractSyntacticSingleKeywordRecommender
+    {
+        public DataKeywordRecommender()
+            : base(SyntaxKind.DataKeyword)
+        {
+        }
+
+        protected override bool IsValidContext(int position, CSharpSyntaxContext context, CancellationToken cancellationToken)
+        {
+            return
+                context.IsGlobalStatementContext ||
+                context.TargetToken.IsUsingKeywordInUsingDirective() ||
```

Comment:
why this line?

Reply:
Good catch, I'll remove. This is a bad carry-over from the `static` recommender I started from.